test/fixtures/component-playground/proxies.js

```diff
@@ -29,6 +29,12 @@ const MarkupProxy = React.createClass({
 });
 
 module.exports = _.merge({}, selectedFixture, {
-  proxies: [PropMutatorProxy, MarkupProxy],
+  proxies: [{
+    component: PropMutatorProxy,
+    props: {},
```

Comment:
This is odd. Isn't `proxies` a prop of the Component Playground and not a fixture property?